package.json

```diff
@@ -93,6 +95,8 @@
     "@mui/material": "^5.11.5",
     "@mui/styled-engine": "^5.11.0",
     "@mui/styled-engine-sc": "^5.11.0",
+    "@storybook/addon-controls": "^6.5.16",
+    "@storybook/addons": "^6.5.16",
```

Comment:
Why do we need this here ?

Reply:
Move to dev dependencies

Reply:
Storybook did that by default i think , i'll try to move it to dev and test.